validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.      raster.stamp_isoscape(filename, "RECALL", recall)
</pre><p>you are missing AUC here.</p><p>I think the code is more robust eliminating this method and changing the code above to select the proper radius key.  then you'll be sure the two (max and per-radius) stay in sync.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>